docs/reference/mapping/types/ip.asciidoc

```diff
@@ -36,6 +36,8 @@ GET my_index/_search
 // CONSOLE
 // TESTSETUP
 
+NOTE: you can also store ip ranges in a single field using an <<range,ip_range datatype>>.
```

Comment:
Nit: Can you start the sentence with an upper case letter? I see this is being done a lot for notes throughout the docs and I think it reads better after rendering.

Reply:
Totally agree, fixed.